addon/templates/components/instructor-selection-manager.hbs

```diff
@@ -25,6 +12,33 @@
     {{/each}}
   </ul>
 {{/if}}
+
+{{#if (get (await instructorGroups) "length")}}
+  <h4>
+    {{t "general.instructorGroups"}}
+    ({{get (await instructorGroups) "length"}})
+  </h4>
+  <div data-test-instructor-groups>
+    {{#each (sort-by "title" (await instructorGroups)) as |instructorGroup|}}
+      <div class="instructor-group" data-test-instructor-group>
+        <span role="button" {{action "removeInstructorGroup" instructorGroup}} data-test-instructor-group-title>
```

Comment:
Can this just be a `<button>`?

Reply:
functionally, yes. but not style-wise. buttons would rewire more style overrides on top. i'd rather not.

![Selection_062](https://user-images.githubusercontent.com/1410427/66792227-a914a980-eeac-11e9-8ca5-843e375e3f73.png)